variables.tf

```diff
@@ -330,6 +330,11 @@ variable "cloudsql_acl_enabled" {
   default     = "true"
 }
 
+variable "config_validator_enabled" {
```

Comment:
These new variables needed to be [forwarded](https://github.com/terraform-google-modules/terraform-google-forseti/blob/master/main.tf#L202) from the root module to the server submodule, otherwise these top level variables will be ignored.

Reply:
I think those 2 variables are listed there in main.tf, is there any other places i need to change to make the forwarding happen?

Reply:
I believe that you were forwarding the variables correctly, my mistake!